app/controllers/application_controller.rb

```diff
@@ -54,7 +54,11 @@ def handle_api_request
 
   def get_user_collections
     if can? :manage, Admin::Collection
-      Admin::Collection.all
```

Comment:
I think this could be left with Admin::Collection.all since the ability to manage Admin::Collection as a class instead of an instance of the class implies all instances.